### PR TITLE
[ticket/16308] Fix UCP reset password

### PR DESCRIPTION
**Pull request by 3D-I**
PHPBB3-16308

Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket:

https://tracker.phpbb.com/browse/PHPBB3-16308

### CODE REVIEW

**Comment by 3D-I** [comment]
!set WIP

**Comment by 3D-I** [comment]
@AlfredoRamos thx, still wip due to something else I want to explore first.

**Comment by 3D-I** [comment]
!unset WIP

**Comment by 3D-I** [comment]
I am done.

File: phpBB/styles/prosilver/template/ucp_reset_password.html [hunk 1]
```diff
@@ -10,7 +10,7 @@ <h2>{{ lang('RESET_PASSWORD') }}</h2>
 
 		<fieldset>
 		{% if S_IS_PASSWORD_RESET %}
-			{% if PASSWORD_RESET_ERRORS %}<p class="error">{{ PASSWORD_RESET_ERRORS | join('<br>') }}</p>{% endif %}
+			{% if PASSWORD_RESET_ERRORS %}<p class="error">{{ PASSWORD_RESET_ERRORS | join('<br>') | raw }}</p>{% endif %}
```
Comment (marc1706) [inline phpBB/styles/prosilver/template/ucp_reset_password.html]: Can you explain why you need `raw`?

Comment (marc1706) [reply phpBB/styles/prosilver/template/ucp_reset_password.html]: Cause it looks fine without it:
image

Comment (3D-I) [reply phpBB/styles/prosilver/template/ucp_reset_password.html]: I have encountered a situation where without `raw` rendered as `<br>`, unable to replicate it at the moment though, moreover... the following 3 examples can be used, I opted for that one.

```
join('<br>')
join('<br>') | raw
join('\n') | nl2br
```

Comment (marc1706) [reply phpBB/styles/prosilver/template/ucp_reset_password.html]: Please remove raw then.

**Pull request merged**